cartography/intel/gcp/compute.py

```diff
@@ -147,6 +227,171 @@ def load_gcp_instances(neo4j_session, data, gcp_update_tag):
             Hostname=instance.get('hostname', None),
             gcp_update_tag=gcp_update_tag
         )
+        _attach_gcp_nics(neo4j_session, instance, gcp_update_tag)
+
+
+def load_gcp_vpcs(neo4j_session, vpcs, gcp_update_tag):
+    """
+    Ingest VPCs to Neo4j
+    :param neo4j_session: The Neo4j session object
+    :param vpcs: List of VPCs to ingest
+    :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    :return: Nothing
+    """
+    query = """
+    MATCH(p:GCPProject{id:{ProjectId}})
+    MERGE(vpc:GCPVpc{id:{PartialUri}})
+    ON CREATE SET vpc.firstseen = timestamp()
+    SET vpc.partial_uri = {PartialUri},
```

Comment:
If we base the ID on PartialUri, we should move it to ON CREATE SET as it will never change and should be added once.